v2/pages/index/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { useContext } from 'react';
+import Link from 'next/link';
+import Head from 'next/head';
+import AuthContext from '../../context/auth';
+import { Heading } from '../../components/UI';
+import IntroVideo from './introVideo';
+import eyecatcher from '../../assets/img/home/eyecatcher.jpg';
+import whatsNew from '../../assets/img/home/wat-is-er-nieuw.png';
+import Download from './download';
+import Cards from './cards';
+import { colors, breakpoints } from '../../assets/styles/constants';
+
+const Home = () => {
+  const { isUserSignedIn, currentUser } = useContext(AuthContext);
+  return (
+    <div className="scroll-snap">
+      <Head>
+        <title>Welkom | Gentlestudent</title>
+      </Head>
+      <section id="one">
+        <Heading level={1} color={colors.white} title="Aan de slag met Gentlestudent" />
+        <Heading level={2} color={colors.white} title="“Verken je stad, help je buren.”" />
+        <IntroVideo />
+      </section>
+      <section id="two">
+        <Heading level={2} color={colors.gray} title="Leerkansen" />
+        <Cards />
+        <Link href="/opportunities">
+          <a className="see-more">Meer leerkansen</a>
+        </Link>
+      </section>
+      <section id="three">
+        <div className="whats-new">
+          <div className="content">
+            <Heading
+              level={2}
+              color={colors.white}
+              title="
+            Zie wat Gentlestudent heeft te bieden en wat je mag verwachten in de toekomst"
+            />
+            <a href="#">Wat is er nieuw?</a>
+          </div>
+        </div>
+        <div className="spacer" />
```

Comment:
Can we give `.content` a `margin-bottom: 10rem;` or give the parent a height, instead of introducing a spacer element with a height?